extensions/sql-database-projects/src/projectProvider/projectProvider.ts

```diff
@@ -104,6 +104,10 @@ export class SqlDatabaseProjectProvider implements dataworkspace.IProjectProvide
 		return Project.openProject(projectFilePath);
 	}
 
+	public promptAddItem(project: sqldbproj.ISqlProject, relativeFilePath: string, itemType?: string): Promise<void> {
```

Comment:
doc comment here too please

Reply:
This is an implementation of the IExtension interface so it picks up doc comments from that